src/averageAges.js

```diff
@@ -14,12 +14,21 @@
  *
  * @return {number}
  */
-function calculateMenAverageAge(people, century) {
+function calculateMenAverageAge(people, century = 0) {
```

Comment:
No need to apply default 0 here, theoretically, century can be 0.
Later you can check if `century === undefined`